packages/documentation-spa/blog/2021-08-09-state-of-one-platform-jul-2021.md

```diff
@@ -0,0 +1,97 @@
+---
+slug: state-of-one-platform-jul-2021
+title: State of One Platform - Jul 2021
+author: Ghanshyam Lohar
+author_title: Senior Software Engineer @ Red Hat
+author_url: https://github.com/ghanlohar
+author_image_url: https://avatars.githubusercontent.com/u/5575651?s=400&v=4
+tags: [redhat, blog, oneportal, oneplatform, react, angular, vue, jul, update]
+---
+### Meta
+**Completed Sprints:** OP – Jul'21 Mid Sprint, OP – Jul'21 End Sprint
+
+**Ongoing/Upcoming Sprint:** OP – Aug'21 Mid Sprint
+
+**One Platform:** [one.redhat.com](https://one.redhat.com)
+
+**JIRA:** [https://projects.engineering.redhat.com/projects/ONEPLAT/issues](https://projects.engineering.redhat.com/projects/ONEPLAT/issues)
+
+**GitHub:** [https://github.com/1-Platform/one-platform/issues](https://github.com/1-Platform/one-platform/issues)
+
+### Introduction
+This is a summary of the development goals achieved by the _One Platform team_ in the month of **July** 2021.
+
+_This month, the strategy for the team was changed and the 4 weeks Sprint cycle was reduced to  2 Weeks cycle_.  As part of two Sprints in July’21, One Platform Team had the following high-level goals which have been achieved as part of **113** **Jira** Issues closed by the Team.
+
+* **Application Development**:
+    * _[Home Page](https://one.redhat.com/) Enhancements_
+        * Rebranding the home page as _Application hub._
+        * Contact us page is updated with new team members data.
+    * _Search_
+        * Indexing native microservice data to Solr - Blocked testing on pre prod environments due to whitelisting of preprod environments in Akamai.
+    * _API Catalog_
+        * Collection of [Requirement](https://docs.google.com/spreadsheets/d/1Flk5hZRM7krzPInlTtfUGnJTN5ktfbyhonKAhEXxMiE/edit#gid=0)
+        * Workflow creation to manage API Catalog with features like graphQL Playground/RESTful API Viewer/Swagger API Viewer
+    * _Component Catalog _
+        * Unified catalog listing component across Patternfly, OPC and Chapeaux
+        * Technical conversation in progress. Requirement documents cover use cases for Phase I development.
+    * _Innovation Hub_
+        * UI development of Landing page.
+        * UI implementation of Idea details page (title, description, total votes, option to vote etc.).
+        * Create & update functionality fields for an Idea such as - Idea Summary and Description (user impact, value, audiences etc.).
+        * _Production release tentatively on 31st Aug '21 date._
+    * _Designs_
+        * Re-defined the Designs for Search SPA, Notification SPA and Banner platform.
+    * _Continuous Integration & Deployment (CI/CD) automation - _with pipeline for each in-built SPAs such as Notification SPA, User SPA, Feedback SPA etc.
+    * Continuing SPAship operator onboarding for individual SPAs.
+* **Platform Development**
+    * _[Developer Console ](https://stage.one.redhat.com/console/)_
```

Comment:
```suggestion
    * _[Developer Console ]_
```
Do we need an internal link?
But that also begs the question about the document (google doc) link, anyone outside redhat won't be able to access it.